npm/npm_import.bzl

```diff
@@ -174,6 +175,12 @@ def npm_import(
             the `direct` attribute of the generated npm_link_package.
         run_lifecycle_hooks: If true, runs `preinstall`, `install` and `postinstall` lifecycle hooks declared in this
             package.
+        lifecycle_hooks_env: Environment variables applied to the lifecycle hooks scripts.
+            Such as some package can define a PREBUILT_BINARY environment variable to download a os specific binary file
+            directly instead of build locally.
+            You can define multiple key/value pairs by providing an "key=value" format array, for example:
```

Comment:
"Lifecycle hooks are defined by providing providing an array of "key=value" entries".

Then maybe move the full example to the end. Something such as:

"
Lifecycle hooks are defined by providing providing an array of "key=value" entries.
For example, defining a `PREBUILT_BINARY`:

`
lifecycle_hooks_env: [
"PREBULT_BINARY=https://downloadurl"],
]
`
"

Reply:
Got it, updated.